include/OpenSpaceToolkit/Astrodynamics/Flight/SatelliteSystem.hpp

```diff
@@ -0,0 +1,100 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/// @project        Open Space Toolkit ▸ Astrodynamics
+/// @file           OpenSpaceToolkit/Astrodynamics/Flight/SatelliteSystem.hpp
+/// @author         Antoine Paletta <antoine.paletta@loftorbital.com>
+/// @license        Apache License 2.0
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#ifndef __OpenSpaceToolkit_Astrodynamics_Flight_SatelliteSystem__
+#define __OpenSpaceToolkit_Astrodynamics_Flight_SatelliteSystem__
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+#include <OpenSpaceToolkit/Mathematics/Objects/Vector.hpp>
+#include <OpenSpaceToolkit/Mathematics/Geometry/3D/Objects/Cuboid.hpp>
+
+#include <OpenSpaceToolkit/Physics/Units/Mass.hpp>
+
+#include <OpenSpaceToolkit/Core/Types/String.hpp>
+#include <OpenSpaceToolkit/Core/Types/Real.hpp>
+#include <OpenSpaceToolkit/Core/Types/Integer.hpp>
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+namespace ostk
+{
+namespace astro
+{
+namespace flight
+{
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+using ostk::core::types::Real ;
+using ostk::core::types::String ;
+using ostk::core::types::Integer ;
+
+using ostk::math::obj::Matrix3d ;
+using ostk::math::geom::d3::objects::Cuboid ;
+
+using ostk::physics::units::Mass ;
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/// @brief                      Defines the dynamics system who's motion is being studied, in particular this a satellite system
+
+class SatelliteSystem
+{
+
+    public:
+
+                                SatelliteSystem                             (   const   Mass&                       aMass,
+                                                                                const   Matrix3d&                   anInertiaTensor,
```

Comment:
@Derollez @robinpdm do you know an easy way to defualt the inertia tensor and satellite geometry here?